subworkflows/local/alignment.nf

```diff
@@ -13,17 +13,17 @@ include { SAMTOOLS_VIEW        } from '../../modules/nf-core/samtools/view/main'
 
 workflow ALIGNMENT {
     take:
-        reads
-        star_index
-        gtf
-        platform
-        subsample_bed
-        seed_frac
-        num_reads
-        switch_subsample_region
-        switch_downsample
-        salmon_index
-        ch_genome_fasta
+        reads                   // channel: [mandatory] [ val(meta), [path(reads)]  ]
+        star_index              // channel: [mandatory] [ val(meta), path(star_index) ]
+        ch_gtf                  // channel: [mandatory] [ val(meta), path(gtf) ]
+        platform                // channel: [ val(platform) ]
+        subsample_bed           // patrameter: [ path(subsample_bed) ]
+        seed_frac               // patrameter: [ val(seed_frac) ]
+        num_reads               // patrameter: [ val(num_reads) ]
+        switch_subsample_region // patrameter: [ val(switch_subsample_region) ]
+        switch_downsample       // patrameter: [ val(switch_downsample) ]
+        salmon_index            // channel: [mandatory] [ path(salmon_index) ]
+        ch_genome_fasta         // channel: [mandatory] [ val(meta), path(fasta) ]
```

Comment:
* platform is currently an ArrayList (`[i, l, l, u, m, i, n, a]`)

* subsample_bed is currently a String, which also means I can't supply my own bed-file: ``` Not a valid path value: '../../../test_data.bed'```

I saw you changed from `// string:` to ` // parameter:` in `ANNOTATE_SNV` so maybe its more of a style preference, but for me that don't know the ins and outs of tomte it's nice to quickly see if the parameter is a boolean, string or number in the comments. 

```suggestion
        reads                   // channel: [mandatory] [ val(meta), [path(reads)]  ]
        star_index              // channel: [mandatory] [ val(meta), path(star_index) ]
        ch_gtf                  // channel: [mandatory] [ val(meta), path(gtf) ]
        platform                // ArrayList: platform 
        subsample_bed           // string: subsample_bed 
        seed_frac               // bigdecimal: seed_frac 
        num_reads               // integer: num_reads
        switch_subsample_region // boolean: switch_subsample_region 
        switch_downsample       // boolean: switch_downsample
        salmon_index            // channel: [mandatory] [ path(salmon_index) ]
        ch_genome_fasta         // channel: [mandatory] [ val(meta), path(fasta) ]
```

Reply:
You are right, it looks better like this :)

Reply:
Ah, I realised that there is something wrong with the way I actually created the ch_platform, I will correct it in the next PR

Reply:
Would be nice to have `subsample_bed` as a channel already, could be created here for example https://github.com/genomic-medicine-sweden/tomte/blob/86de5040d712757d6bebf8d6e9342515cdcdc0de/workflows/tomte.nf#L56

We can leave it for discussion and a later PR